### ISSUE TITLE
Valid loss explode after several epochs

### ISSUE THREAD

**Issue by Hugh0120** [zzh8829/yolov3-tf2#44]
![image](https://user-images.githubusercontent.com/13382771/62301602-7eb96f80-b4ab-11e9-9824-0fc706b91a35.png)
here is the picture of my training procedure, after several epochs' of decreasing, the valid loss suddenly exploded, did it happened to you when you were training?

**Comment by AnaRhisT94** [comment]
@Hugh0120  Hi, how did you get the model to train in the first place? I'm still struggling.
About the valid loss that suddenly exploded, try to lower the LR by a factor of 10 when getting to 20 epochs.

**Comment by Hugh0120** [comment]
> @Hugh0120 Hi, how did you get the model to train in the first place? I'm still struggling.
> About the valid loss that suddenly exploded, try to lower the LR by a factor of 10 when getting to 20 epochs.

I just trained the model from scratch

**Comment by Hugh0120** [comment]
> @Hugh0120 Hi, how did you get the model to train in the first place? I'm still struggling.
> About the valid loss that suddenly exploded, try to lower the LR by a factor of 10 when getting to 20 epochs.

btw, I change the code from keras to tf estimator and the loss decreased properly

**Comment by sedrasmith** [comment]
@Hugh0120 for me the validation loss is exploding from the beginning. disable the ReduceLROnPlateau(verbose=1) in keras callback will do the job.
